sgv2-graphqlapi/src/main/java/io/stargate/sgv2/graphql/schema/Uuids.java

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.graphql.schema;
+
+import java.lang.management.ManagementFactory;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Enumeration;
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Random;
+import java.util.Set;
+import java.util.TimeZone;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Provides utility methods to handle time-based UUIDs.
+ *
+ * <p>Adapted from the <a
+ * href="https://github.com/datastax/java-driver/blob/4.x/core/src/main/java/com/datastax/oss/driver/api/core/uuid/Uuids.java">Java
+ * driver class of the same name</a> (also under Apache v2 license).
+ */
+public class Uuids {
```

Comment:
While this is fine and probably copied from earlier impl, was curious if we really need our own UUID generator implementation? Could just use `java-uuid-generator` (https://mvnrepository.com/artifact/com.fasterxml.uuid/java-uuid-generator)  for time-based generation it seems, unless there are differences from standard UUID mechanism?

Reply:
Yes, this was copied from the Java driver.
I don't have a strong preference, the only advantage of is to avoid yet another dependency, but the JAR is pretty small so maybe it doesn't matter that much.
Do you see anything that JUG does better/differently?

Reply:
I did not notice anything special so that is fine: especially since it comes from an existing implementation that appears to be working fine. Who knows, maybe it originally comes from JUG or one of alternatives (code looked familiar but that could just be because it does the same thing).

So this is good then.